### DISCUSSION TITLE
Help me with ROLLUP function

### DISCUSSION THREAD

**Discussion by mustard428** [dbeaver/dbeaver#15667] [Q&A]
Hello.

I'd like to use ROLLUP function with SQL, but I got the message below.

SQL Error [1]: [SQLITE_ERROR] SQL error or missing database (no such function: ROLLUP)

What should I check? Other Queries works well I guess.

Thank you for your help.

**Comment by TheMechanist902** [comment]
It looks like SQLite doesn't support the ROLLUP function. At least I can't find any mention of it in [it's documentation](https://www.sqlite.org/docs.html) and [this SO](https://stackoverflow.com/questions/21732281/sqlite-subtotals-in-own-row-aka-rollup) topic just gives some workaround advise.
If you want to use the function you would need to use an DBMS that support it e.g. Postgres.

Please notice that this is not a dBeaver related problem, but a a problem of the used backend. So you might be getting more help easier on specific platforms for those DBMS.
